.travis.yml

```diff
@@ -6,6 +6,12 @@ node_js:
   - 10
   - 11
 
-script: npm run travis
+matrix:
+  include:
+    - name: Lint
+      node_js: 12
+      script: npm run lint
```

Comment:
Prettier 2 doesn't support node 8, so add a build matrix so we only lint with node 12.